plugins/cuda/cuda_plugin.c

```diff
@@ -565,6 +565,12 @@ int cuda_plugin_init(int stage)
 {
 	int ret;
 
+	/* Disable CUDA checkpointing with pre-dump */
+	if (stage == CR_PLUGIN_STAGE__PRE_DUMP) {
```

Comment:
Form this comment and from the commit message, it is unclear why we need to disable the plugin.